schema.graphql

```diff
@@ -361,3 +361,25 @@ type Deposit @entity {
   " pool where the funds are deposited "
 	pool: PoolCreated!
 }
+
+type DealSponsored @entity {
+	" poolAddress + sponsorAddress"
+	id: ID!
+
+  " the address of the sponsor "
+	userAddress: Bytes!
```

Comment:
again maybe we want to go with sponsor instead of userAddress, but its not a big deal